public/locales/nl/common.json

```diff
@@ -153,12 +153,14 @@
   "modules/media/const/index___bookmark": "Bookmark",
   "modules/media/const/index___creatiedatum": "Creatiedatum",
   "modules/media/const/index___dit-formaat-wordt-niet-ondersteund-format": "Het \"{{format}}\"-formaat wordt niet ondersteund.",
+  "modules/media/const/index___dit-object-bevat-geen-licentie-voor-online-raadpleging-breng-een-bezoek-aan-de-leeszaal": "Dit object bevat geen licentie voor online raadpleging. Breng een bezoek aan de leeszaal.",
   "modules/media/const/index___duurtijd": "Duurtijd",
   "modules/media/const/index___episode": "Episode",
   "modules/media/const/index___er-ging-iets-mis-bij-het-ophalen-van-het-afspeel-token": "Er ging iets mis bij het ophalen van het afspeel-token.",
   "modules/media/const/index___filmbasis": "Filmbasis",
   "modules/media/const/index___genre": "Genre",
   "modules/media/const/index___het-formaat-van-de-data-wordt-op-dit-moment-niet-ondersteund": "Het formaat van de data wordt op dit moment niet ondersteund.",
+  "modules/media/const/index___je-kan-het-object-enkel-bekijken-tijdens-een-fysiek-bezoek-aan-de-leeszaal": "Je kan het object enkel bekijken tijdens een fysiek bezoek aan de leeszaal.",
```

Comment:
```suggestion
  "modules/media/const/index___je-kan-het-object-enkel-bekijken-tijdens-een-fysiek-bezoek-aan-de-leeszaal": "Je kan dit object enkel bekijken tijdens een fysiek bezoek aan de bezoekersruimte.",
```

Reply:
bij voorkeur ook de translation key fixen in zowel de common.js file als in de .ts file